sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -442,16 +441,16 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
       val expected = sql("SELECT key FROM small_src").collect().head
       val res = sql(
         """
-        |SELECT TRANSFORM (key) ROW FORMAT SERDE
-        |'org.apache.hadoop.hive.serde2.avro.AvroSerDe'
-        |WITH SERDEPROPERTIES ('avro.schema.literal'='{"namespace":
-        |"testing.hive.avro.serde","name": "src","type": "record","fields":
-        |[{"name":"key","type":"int"}]}') USING 'cat' AS (tKey INT) ROW FORMAT SERDE
-        |'org.apache.hadoop.hive.serde2.avro.AvroSerDe' WITH SERDEPROPERTIES
-        |('avro.schema.literal'='{"namespace": "testing.hive.avro.serde","name":
-        |"src","type": "record","fields": [{"name":"key","type":"int"}]}')
-        |FROM small_src
-      """.stripMargin.replaceAll(System.lineSeparator(), " ")).collect().head
+          |SELECT TRANSFORM (key) ROW FORMAT SERDE
+          |'org.apache.hadoop.hive.serde2.avro.AvroSerDe'
+          |WITH SERDEPROPERTIES ('avro.schema.literal'='{"namespace":
+          |"testing.hive.avro.serde","name": "src","type": "record","fields":
+          |[{"name":"key","type":"int"}]}') USING 'cat' AS (tKey INT) ROW FORMAT SERDE
+          |'org.apache.hadoop.hive.serde2.avro.AvroSerDe' WITH SERDEPROPERTIES
+          |('avro.schema.literal'='{"namespace": "testing.hive.avro.serde","name":
+          |"src","type": "record","fields": [{"name":"key","type":"int"}]}')
+          |FROM small_src
+        """.stripMargin.replaceAll(System.lineSeparator(), " ")).collect().head
```

Comment:
It seems like this is all related to formatting?"

Reply:
Yeah

Reply:
I have already added this to the `description` of PR: `beautify some code formats.`

Reply:
I have restored the changes that were only `formatted` in the code.